manifests/b/Badlion/BadlionClient/3.2.0/Badlion.BadlionClient.yaml

```
@@ -4,4 +4,4 @@ PackageIdentifier: Badlion.BadlionClient
 PackageVersion: 3.2.0
 DefaultLocale: en-US
 ManifestType: version
-ManifestVersion: 1.0.0
+ManifestVersion: 1.0.1
```

```suggestion
ManifestVersion: 1.0.0

```
This key must have the value "1.0.0". The Microsoft community package repository validation pipelines also use this value to determine appropriate validation rules when evaluating this file.

---

Thanks for the advice. I wasn't aware that it changed the validation rules